app/services/fhir/resources/fillers.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any, Dict
+
+
+def create_code() -> Dict[str, Any]:
+    return {
+        "system": "System not available",
+        "text": "system was added during update cycle to bypass validation",
+    }
+
+
+def fill_pracitioner_qualification(qualification: Dict[str, Any]) -> Dict[str, Any]:
+    if "code" not in qualification:
+        qualification["code"] = create_code()
+
+    return qualification
+
+
+def fill_location_position(position: Dict[str, Any]) -> Dict[str, Any]:
+    if "longitude" not in position:
+        position["longitude"] = 0.0
+
+    if "latitude" not in position:
+        position["latitude"] = 0.0
+
+    return position
+
+
+def fill_endpoint(data: Dict[str, Any]) -> Dict[str, Any]:
+    if "status" not in data:
+        data["status"] = "error"
```

Comment:
See [link](https://hl7.org/fhir/R4B/valueset-endpoint-status.html#expansion)
```suggestion
        data["status"] = "test"
```